sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/yaml/YamlToInternalMappersTest.java

```diff
@@ -0,0 +1,44 @@
+package com.mesosphere.sdk.specification.yaml;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * This class tests {@link YAMLToInternalMappers}.
+ */
+public class YamlToInternalMappersTest {
+    private static final String firstUriStr = "https://github.com/mesosphere/dcos-commons/blob/master/README.md";
+    private static final String secondUriStr = "https://github.com/mesosphere/dcos-commons/blob/master/docs/generate.sh";
+    private static final String multipleSpacedUriStr = firstUriStr + ", " + secondUriStr;
+    private static final String multipleNoSpaceUriStr = firstUriStr + "," + secondUriStr;
+
+    @Test(expected = URISyntaxException.class)
+    public void testEmptyUriString() throws URISyntaxException {
```

Comment:
Probably overkill for this kind of method, but we could also test the following cases:
- `" "`
- `","`
- `firstString + ","`
- `"," + secondString`